src/Icons/src/Command/WarmCacheCommand.php

```diff
@@ -41,16 +39,14 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $io = new SymfonyStyle($input, $output);
         $io->comment('Warming the icon cache...');
```

Comment:
In WarmIconCacheOnAssetCompileListener we already display this same comment ... maybe this is oputput twice then?

Reply:
This shouldn't be an issue as they are two different things:
1. `ux:icons:warm-cache` is a standalone cli command
2. `WarmIconCacheOnAssetCompileListener` is only used with `asset-map:compile`